net-mgmt/zabbix-proxy/src/opnsense/mvc/app/models/OPNsense/Zabbixproxy/General.xml

```diff
@@ -79,5 +79,24 @@
             <default>4</default>
             <Required>N</Required>
         </timeout>
+        <encryption type="BooleanField">
+            <default>0</default>
+            <Required>Y</Required>
+            <advanced>true</advanced>
```

Comment:
@fzoske oh, @mimugmail is right. advanced is for the form, not the model

last bit before merge... many thanks so far!